src/Core/Settings/GlobalSettings.cs

```diff
@@ -260,8 +260,31 @@ public string JobSchedulerConnectionString
 
     public class EventLoggingSettings
     {
+        public AzureServiceBusSettings AzureServiceBus { get; set; } = new AzureServiceBusSettings();
+        public virtual string HttpPostUrl { get; set; }
         public RabbitMqSettings RabbitMq { get; set; } = new RabbitMqSettings();
 
+        public class AzureServiceBusSettings
+        {
+            private string _connectionString;
+            private string _topicName;
+
+            public virtual string EventRepositorySubscriptionName { get; set; } = "events-write-subscription";
+            public virtual string HttpPostSubscriptionName { get; set; } = "events-httpPost-subscription";
```

Comment:
```suggestion
            public virtual string HttpPostSubscriptionName { get; set; } = "events-http-post-subscription";
```

⛏️ but can you also update all the docs and wherever else this is used? This seems to be a more traditional pattern for the actual eventing.